js/change-list.js

```diff
@@ -175,8 +250,14 @@ class ChangeList {
       // console.log();
       // console.log(OP_TABLE[op].name);
       // console.log(this.stack);
-      i = OP_TABLE[op](this.stack, mem8, mem32, i);
+      i = OP_TABLE[op](this.stack, this.eventsTrampoline, this.listeners, mem8, mem32, i);
```

Comment:
Random stylistic question, should the table take `this` instead of the fields of `this`?

Reply:
Perhaps! I just kept adding one more thing at a time... By now, it likely should just take `this`!

It is also *possible* that passing the pieces is faster if the `this` properties don't end up monomorphic in the JIT's eyes, but that (a) seems unlikely given, and (b) shouldn't matter relative to the DOM method costs / is definitely premature optimization.